inttest/bootloose-alpine/Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN apk add --no-cache \
   haproxy \
   nginx \
   inotify-tools \
-  ipvsadm
+  ipvsadm \
+  jq
```

Comment:
I don't see where this is required?

Reply:
Not required per se, but makes life easier to have it in place when needing to debug things with etcd and it's various status&health outputs